cmd/rook/nfs/server.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2018 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nfs
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/controller"
+	"github.com/rook/rook/cmd/rook/rook"
+	"github.com/rook/rook/pkg/operator/nfs"
+	"github.com/rook/rook/pkg/util/flags"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+var serverCmd = &cobra.Command{
+	Use:   "server",
+	Short: "Runs the NFS server to deploy and manage NFS server in kubernetes clusters",
+	Long: `Runs the NFS operator to deploy and manage NFS server in kubernetes clusters.
+https://github.com/rook/rook`,
+}
+
+var (
+	provisioner, server, path, ganeshaConfigPath *string
+)
+
+func init() {
+	flags.SetFlagsFromEnv(serverCmd.Flags(), rook.RookEnvVarPrefix)
+	flags.SetLoggingFlags(serverCmd.Flags())
+
+	provisioner = serverCmd.Flags().String("provisioner", "", "Name of the provisioner. The provisioner will only provision volumes for claims that request a StorageClass with a provisioner field set equal to this name.")
+	server = serverCmd.Flags().String("server", os.Getenv("ROOK_NFS_SERVICE_HOST"), "Name of the NFS server")
+	ganeshaConfigPath = serverCmd.Flags().String("ganeshaConfigPath", "", "ConfigPath of nfs ganesha")
+
+	serverCmd.RunE = startServer
+}
+
+func startServer(cmd *cobra.Command, args []string) error {
```

Comment:
is this essentially running in golang code everything that was being done in the `start.sh` script inside the container image before?

Reply:
yeah golang not executes the start.sh commands